src/logger/index.ts

```diff
@@ -20,28 +27,35 @@ const logLevelMapping = {
 
 const LOG_LEVEL = config.logger.level && logLevelMapping[config.logger.level] || LogLevel.important;
 const LOG_TO_FILES = config.logger.logToFiles || false;
+// LOG_PATH defaults to `ceramic-anchor-service/logs/`
 let LOG_PATH = config.logger.filePath || path.join(path.resolve(__dirname, '../../'), 'logs/');
 if (!LOG_PATH.endsWith('/')) {
   LOG_PATH = LOG_PATH + '/';
 }
 
-const ACCESS_FILE_PATH = LOG_PATH + 'access.log';
-const METRICS_FILE_PATH = LOG_PATH + 'metrics.log';
-const EVENTS_FILE_PATH = LOG_PATH + 'events.log';
+const ACCESS_FILE_PATH = path.join(LOG_PATH, 'access.log');
+const EVENTS_FILE_PATH = path.join(LOG_PATH, 'events.log');
+const METRICS_FILE_PATH = path.join(LOG_PATH, 'metrics.log');
+const STDOUT_FILE_PATH = path.join(LOG_PATH, 'stdout.log');
```

Comment:
Can we call this diagnostics.log/DIAGNOSTICS_FILE_PATH?  That way we're describing what the content represents rather than one place it might be sent sometimes.  It feels especially weird to call this stdout.log when if you're logging to files and haven't set log verbosity to 'debug', there's no stdout output

Reply:
There is stdout on log levels other than debug